internal/document/editor/editorservice/service_test.go

```diff
@@ -95,7 +95,7 @@ func Test_IdentityUnspecified(t *testing.T) {
 			assert.True(t, ok)
 			assert.Len(t, dResp.Notebook.Metadata, 2)
 			assert.Contains(t, rawFrontmatter, "prop: value\n")
-			assert.Contains(t, rawFrontmatter, "id: 123\n")
+			assert.Contains(t, rawFrontmatter, "id: \"123\"\n")
```

Comment:
Important: It's interesting why it changed. It should be debugged.

Reply:
This is odd indeed, it appears that `abc123` won't require the quotes whereas a number `123` does. Wondering if the YAML type serialization quotes all numbers when casting to string?

Reply:
Yes, if a number is treated as a string, it should be quoted. 

However, in this case, I changed the logic and used formatted `RunmeMetadata` always. If we don't require identity, the raw value should be used. It's fixed in https://github.com/stateful/runme/pull/431/commits/e58c2060567aacf38b9d2eb6d252d84dd60ed02e.